src/constructs/bindings.md

```diff
@@ -2,16 +2,19 @@
 title: Bindings
 ---
 
-## On the Subject of `let` and `let rec`
+## On the Subject of `let`, `let rec`, and `let mut`
 
 The `let` and `let rec` statements introduce new names into the current scope.
 In other words, they let you declare new bindings (quite similar to constants in other languages).
 
+The `let mut` statement can be used to introduce new mutable bindings into the current scope.
+
 ## Syntax
 
 ```grain
 let name1 = value1 [, name2 = value2]
 let rec name1 = func1 [and name2 = func2]
+let mut name1 = value1 [, name2 = value2]
```

Comment:
I realized that we don't at all mention destructuring here. Should we open an issue for that?

Reply:
Will do!